assets/src/edit-story/components/canvas/displayElement.js

```diff
@@ -59,21 +69,26 @@ function DisplayElement({ element }) {
       target.style.transform = '';
       target.style.width = '';
       target.style.height = '';
+      target.style.opacity = 1;
     } else {
-      const { translate, rotate, resize } = transform;
-      target.style.transform = `translate(${translate[0]}px, ${translate[1]}px) rotate(${rotate}deg)`;
-      if (resize[0] !== 0 && resize[1] !== 0) {
+      const { translate, rotate, resize, updates } = transform;
+      target.style.transform = `translate(${translate?.[0]}px, ${translate?.[1]}px) rotate(${rotate}deg)`;
+      if (resize && resize[0] !== 0 && resize[1] !== 0) {
         target.style.width = `${resize[0]}px`;
         target.style.height = `${resize[1]}px`;
       }
+      if (updates) {
+        target.style.opacity = updates.overDropTarget ? 0.6 : 1;
```

Comment:
Nit, but I think this is best to be its own property, not "updates". Maybe we can call it `replacement: {overDropTarget, resource, ...}`?